### ISSUE TITLE
JSON-Schema 2019-09 (draft-08) released

### ISSUE THREAD

**Issue by tomByrer** [json-schema-faker/json-schema-faker#532]
I know you're busy, but FYI there is a new draft:
http://json-schema.org/specification.html

I'm not sure what directly affects this lib but here is a few of the [changes](http://json-schema.org/draft/2019-09/release-notes.html#incompatible-changes):

* By default, `format` is no longer an assertion
* Plain name fragments are no longer defined with $id, but instead with the new keyword [$anchor](http://json-schema.org/draft/2019-09/json-schema-core.html#rfc.section.8.2.3) (which has a different syntax).
* `definitions` is now [$defs](http://json-schema.org/draft/2019-09/json-schema-core.html#rfc.section.8.2.5)

Maybe a few more I missed?

I'm just checking out your lib, it seems really neat!  But I'm uncomfortable to make a PR ATT.

**Comment by pateketrueke** [comment]
Yeah, doing the translation on-the-fly does not seems complicated, but it requires time.

I hope fixing those soon, thank you! :beers:
